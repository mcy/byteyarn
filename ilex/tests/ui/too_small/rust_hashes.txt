r#'foo'# r'foo' q###'bar'### q##'bar'##
