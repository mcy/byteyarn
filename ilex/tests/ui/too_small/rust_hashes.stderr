error: unrecognized characters
 --> too_small/rust_hashes.txt:1:10
  |
1 | r#'foo'# r'foo' q###'bar'### q##'bar'##
  |          ^^^^^^
  |
  = note: reported at: ilex/src/rt/mod.rs:36:8

error: unexpected closing `'##`
 --> too_small/rust_hashes.txt:1:37
  |
1 | r#'foo'# r'foo' q###'bar'### q##'bar'##
  |                                     ^^^ expected to be opened by `r##'`
  |
  = note: reported at: ilex/src/rt/emit2.rs:254:22

error: unrecognized characters
 --> too_small/rust_hashes.txt:1:30
  |
1 | r#'foo'# r'foo' q###'bar'### q##'bar'##
  |                              ^^^^^^^
  |
  = note: reported at: ilex/src/rt/mod.rs:36:8

error: aborting due to 3 errors
